packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -352,8 +343,8 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
     }
   }
 
-  private _onPanelClick = (): void => {
-    this.dismiss();
+  private _onPanelClick = (ev?: any): void => {
```

Comment:
Worth typing this as a `SyntheticEvent` also? Both `mouse` and `keyboard` can invoke this handler.